src/Misc/layoutbin/powershell/Add-XamarinAndroidCapabilities.ps1

```diff
@@ -1,4 +1,19 @@
 [CmdletBinding()]
 param()
 
-$null = Add-CapabilityFromRegistry -Name 'Xamarin.Android' -Hive 'LocalMachine' -View 'Registry32' -KeyName 'Software\Novell\Mono for Android' -ValueName 'InstalledVersion'
+if (!(Add-CapabilityFromRegistry -Name 'Xamarin.Android' -Hive 'LocalMachine' -View 'Registry32' -KeyName 'Software\Novell\Mono for Android' -ValueName 'InstalledVersion')) {
+    $vs15 = Get-VisualStudio_15_0
+    if ($vs15 -and $vs15.installationPath) {
+        # End with "\" for consistency with old ShellFolder values.
+        $shellFolder15 = $vs15.installationPath.TrimEnd('\'[0]) + "\"
+        $xamarinAndroidDir = ([System.IO.Path]::Combine($shellFolder15, 'MSBuild', 'Xamarin', 'Android')) + '\'
+        if ((Test-Container -LiteralPath $xamarinAndroidDir)) {
+            $versionFile = ([System.IO.Path]::Combine($xamarinAndroidDir, 'Version'))
+            $version = Get-Content $versionFile
+            if ($version) {
+                Write-Capability -Name 'Xamarin.Android' -Value $version
```

Comment:
The "Version" file seems well formed so I didn't need to trim.  It only has one line.

Also I tested when the file doesn't exist on disk, I didn't observe any ill behaviors.  Do I need to do add (-literalpath or -erroraction ignore)?  I'd like to follow the best practice.